wdl/Module10Benchmark.wdl

```diff
@@ -4,7 +4,7 @@ import "Structs.wdl"
 import "TasksMakeCohortVcf.wdl" as MiniTasks
 import "TasksBenchmark.wdl" as tasks10
 
-import "VaPoR.wdl" as vapor
+import "Vapor.wdl" as vapor
```

Comment:
```suggestion
```

Reply:
Deleted the other unused imports as well